src/main/java/Exercise9.java

```diff
@@ -0,0 +1,12 @@
+public class Exercise9 {
+    public float calculatePi(){
+        int i = 0;
+        float result = 0;
+        while(Math.abs(3.1415 - result) >= 0.0001){
+            if(i % 2 == 0) result += (float) 4/(2 * i + 1 );
+            else result -= (float) 4/(2 * i + 1 );
+            i++;
```

Comment:
Nếu tăng i lên 2 đơn thì có thể bỏ được lệnh if

Reply:
e dùng if để cho phép tính + và - xen kẻ nhau phù hợp với yêu cầu bài toán

Reply:
Done